jf_agent/jf_jira/jira_download.py

```diff
@@ -290,89 +290,106 @@ def download_boards_and_sprints(jira_connection, project_ids, download_sprints):
 @diagnostics.capture_timing()
 @agent_logging.log_entry_exit(logger)
 def download_all_issue_metadata(
-    jira_connection, project_ids, earliest_issue_dt, num_parallel_threads, issue_filter
+    jira_connection, all_project_ids, earliest_issue_dt, num_parallel_threads, issue_filter
 ) -> Dict[int, IssueMetadata]:
     print('downloading issue metadata... ', end='', flush=True)
-    issue_jql = (
-        f'project in ({",".join(project_ids)}) and updatedDate > '
-        f'{"0" if not earliest_issue_dt else earliest_issue_dt.strftime("%Y-%m-%d")}'
-    )
-    if issue_filter:
-        issue_jql += f' and {issue_filter}'
-    total_num_issues = jira_connection.search_issues(issue_jql, fields=['id']).total
-    issues_per_thread = math.ceil(total_num_issues / num_parallel_threads)
+
```

Comment:
This is the section with the changes - the diff is confusing because the for loop changed the spacing for everything below it.